pymc3/distributions/continuous.py

```diff
@@ -132,11 +148,12 @@ class HalfNormal(Continuous):
       - `sd` : sd > 0 (alternative parameterization)
 
     """
-    def __init__(self, tau=None, sd=None, *args, **kwargs):
+    def __init__(self, tau=None, sd=None, transform=logtransform, *args, **kwargs):
```

Comment:
Isn't the transform already specified in `PositiveContinuous` here? Shouldn't have to specify it twice.